python/doc/pyplots/LHSExperiment.py

```diff
@@ -8,5 +8,15 @@
 bounds = distribution.getRange()
 lhs = ot.LHSExperiment(distribution, size)
 sample = lhs.generate()
-subdivisions = [size] * dim
```

Comment:
This line shows that the number of subdivisions is equal to the sample size, which is the essential property of a LHS. Replacing this code by a calculation obscures the code and hides the mathematical feature of this particular design of experiment.